tests/test_utils.py

```diff
@@ -133,6 +134,49 @@ def test_get_editor_precedence():
     assert utils.get_editor(onyo_root) == 'third'
 
 
+def test_generate_faux_serials():
+    """
+    Test the creation of unique serial numbers with the default length through
+    calling the generate_faux_serial() function multiple times and comparing the
+    generated faux serial numbers.
+    """
+    onyo_root = './'
+
+    # technically, creating multiple faux serial numbers without actually
+    # creating assets might lead to duplicates. In practice, this should not
+    # happen if they are sufficiently long and random
+    faux = [utils.generate_faux_serial(onyo_root) for x in range(0, 100)]
+    assert len(faux) == len(set(faux))
+    assert len(faux) > 0
+
+
+def test_generate_faux_serials_different_lengths():
+    """
+    Create many faux serial numbers with different lengths. Because of the
+    different lengths, they should all be different.
+    """
+    onyo_root = './'
```

Comment:
That is true. Should all functions in `utils` that get `onyo_root` test if it is actually the onyo repository root directory? I think it might be better to check this beforehand in the onyo main or onyo commands that call sub-functions from utils.
Otherwise there are a lot of tests that would need to be run for many of the utils function calls.